src/lib/repositories/impl_no_sql/order_status_history_repository_impl.py

```diff
@@ -128,3 +128,22 @@ def get_unprocessed_order_status_histories(self):
 
     def get_last_order_status_histories_by_order_ids(self, order_id):
         pass
+
+    def get_unassigned_order_status_histories(self):
+        unassigned_order_status_histories_from_mongo = OrderStatusHistory.objects(
```

Comment:
In order to re use this method, why don't you  send the service as a parameter? And also I would add a limit

Reply:
This part was move to another PR. https://github.com/eapg/EA_RESTAURANT/pull/231